### DISCUSSION TITLE
Asus Zenbook UM3402 doesn't show CPU temperature and there is no Flicker free dimming option

### DISCUSSION THREAD

**Discussion by valchazzz** [seerge/g-helper#2462] [General]
Hello! My new Asus  Zenbook UM3402YA-KM454W with OLED screen does not have Flicker free dimming option in G-helper app. I have this option in My Asus APP.
And app doesn't show CPU temperature.

Am I missing something or there is not support now for my laptop model?

![image](https://github.com/seerge/g-helper/assets/167644992/131337d5-4bd9-4e11-ac0f-2190f932d7d8)

![image](https://github.com/seerge/g-helper/assets/167644992/b2a8f9d9-9c71-4809-8bdf-94216e66d9f3)

**Comment by seerge** [comment]
@valchazzz hello, this build should show dimming for your model, and also show correctly dropdowns in Visual Mode section . Please confirm

[GHelper.zip](https://github.com/seerge/g-helper/files/15049840/GHelper.zip)

As for CPU temperature, app checks it via known Asus endpoint + checks in windows. If it's not there - it just keeps it blank. Could be limitation of your device.

**Reply by seerge** [reply]
P.S. Try to run app as admin, may be then windows will report temperature

**Reply by valchazzz** [reply]
Thank you, dimming is showing and working with this build, thank you!
